.github/workflows/nightly_tests.yml

```diff
@@ -181,64 +175,7 @@ jobs:
       if: always()
       run: |
         pip install slack_sdk tabulate
-        python scripts/log_reports.py >> $GITHUB_STEP_SUMMARY
-
-  run_lora_nightly_tests:
```

Comment:
We run LoRA tests in the ` Nightly Torch CUDA Tests` job since PEFT is a needed dependency for LoRA loading. We don't need a job dedicated PEFT job anymore.